### PR TITLE
Multi language

### PR DESCRIPTION
**Pull request by TheRealMoeder**
Probably fail to build the pull request, as encrypted variables are not available in pull requests for safety reasons  and also it is not yet defined.

### CODE REVIEW

**Comment by F-D-R** [comment]
What kind of token should I put, and where to?
Would you give me some instructions, or give a link to a document, which describes the process?

**Comment by TheRealMoeder** [comment]
Go to user settings on Transifex, click on API token on the left and generate a new one. Then copy the token to your clipboard. Go to travis-ci.org, select DeviationTX/deviation-manual, edit settings and create a new environment variable named TXPASSWD and paste the token from clipboard as it's value. Safe the environment variable.

**Comment by TheRealMoeder** [comment]
Oh, I just noticed I can do that myself, if you want to...

**Comment by F-D-R** [comment]
I don't see such option in Transifex neither in the organization settings nor in the deviation-manual settings.
Also the project is still linked to your GitHub fork.

**Comment by F-D-R** [comment]
Found it, it was in the user settings...

**Comment by F-D-R** [comment]
Despite I've made the token, the build has failed again...

**Comment by TheRealMoeder** [comment]
I changed the repository in transifex (although it is only needed for verification that it is an open source project).
Like I said above, Travis will _NEVER_ pass, as tokens are not given away for pull request. Otherwise you could change travis.yml to "echo $MY_ULTIMATELY_SECRET_PASSWORD" and it would get revealed by anyone making a pull request. Travis is smart enough to only share its encrypted variables with commits merged to the repository.

You can see everything works in the [travis log for my account](https://www.travis-ci.org/TheRealMoeder/deviation-manual/builds/218508804)

**Comment by F-D-R** [comment]
I think I understand. ;)
So that's why the decryption fails too...

**Comment by F-D-R** [comment]
So tell me how to get the GitHub token into GH_TOKEN.
If it is a "Personal access token", what privilegs does it need, i.e. which checkbox do I need to check?
Is it the repo or the repo_hook, or neither of them?

**Comment by TheRealMoeder** [comment]
Yes it is, you need public_repo "only".

But please note pushing back to github is currently disables, as I'm at war with git and its branching system once again ;)

**Comment by F-D-R** [comment]
Done.

**Review by F-D-R — APPROVED** [review]

**Comment by TheRealMoeder** [comment]
Go ahead and merge if you wish. I'm ready for fixes in case anything happens.

File: source/conf.py-intl [hunk 1]
```diff
@@ -0,0 +1,412 @@
+# -*- coding: utf-8 -*-
```
Comment (TheRealMoeder) [inline source/conf.py-intl:1]: BTW, this is used only for getting the source strings because sphinx-intl currently cannot handle "tag"

**Pull request merged**